### ISSUE TITLE
AriaLiveAnnouncer and alerts work poorly with VoiceOver

### ISSUE THREAD

**Issue by jessegreenberg** [phetsims/utterance-queue#87]
I have an old mac:

macOS Big Sur
11.6.2
Retina, 13-inch, Late 2013
2.4 GHz, Dual-Core Intel i15
Memory 8 GB

And aria-live alerts very rarely work in the sim.

VoiceOver in general very rarely works on this machine, I have to disable the expensive graphics in the sim or I just hear "VoiceOver busy". It may be that this machine is too old for our accessibility needs. But I am worried that alerts in general are not working well because of issues like https://github.com/phetsims/greenhouse-effect/issues/192.

**Comment by jessegreenberg** [comment]
Over slack, @Nancy-Salpepi said that alerts generally come through predictably for VoiceOver on her Mac. I am going to close this and note that VoiceOver just behaves very poorly when the machine is struggling with performance. For my old machine that is almost always, but for up-to-date devices they should be able to keep up with PhET sims and accessibility.

Closing.

**Comment by Nancy-Salpepi** [comment]
Figured I would add a little video even though you closed the issue.



https://user-images.githubusercontent.com/87318828/181368410-b1ae9a7e-316c-4510-8cd6-2a9e335da30d.mov
